### PR TITLE
i18n(ja): Add deploy/google-firebase.mdx

### PR DESCRIPTION
**Pull request by morinokami**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- Add deploy/google-firebase.mdx.

### CODE REVIEW

File: src/pages/ja/guides/deploy/google-firebase.mdx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: AstroサイトをGoogleのFirebase Hostingにデプロイする
+description: GoogleのFirebase HostingによりAstroサイトをウェブにデプロイする方法。
+layout: ~/layouts/DeployGuideLayout.astro
+i18nReady: true
+---
+
+[Firebase Hosting](https://firebase.google.com/products/hosting)は、Googleのアプリ開発プラットフォーム[Firebase](https://firebase.google.com/)が提供するサービスで、Astroサイトをデプロイするために使用可能です。
```
[the remark this answers is not in the document]
Comment (delucis) [reply src/pages/ja/guides/deploy/google-firebase.mdx:8]: Yes! Anywhere you feel different word/sentence order makes more sense in the translation language, please feel free to change the order! We want to preserve meaning, but details like word order are not important.

**Comment by morinokami** [comment]
@pilcrowOnPaper そのままだと息切れするため読点「、」をはさみましたが、提案いただいた内容で更新しました。

**Comment by pilcrowOnPaper** [comment]
LGTM 👍

**Review by delucis — APPROVED** [review]
SIX!

**Pull request merged**